### PR TITLE
Add first version of Themes

### PR DESCRIPTION
**Pull request by MartinNemi03**
So it works like this: 
- Themes are stored in src/constants/themes.js
- When a site is requested, it loads in with extra
- Default theme is "default", automatically selected, when "currentTheme" isn't stored in LocalStorage

Please review and give feedback.
Let's hope this doesn't break.

### CODE REVIEW

**Review by boblovespi — CHANGES_REQUESTED** [review]
themes and packs can be open at the same time

**Comment by MartinNemi03** [comment]
> 
> 
> themes and packs can be open at the same time

i forgot that. ok, gimme a moment.

File: src/constants/themes.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                link: "#0BEA51",
+                hover: "#09EF70",
+                maxed: "#DD980E",
+                gold: "#FDBB3C",
+                skillbar: "#0BAA51",
+                maxedbar: "#CE8F12"
+            }
+        },
+        "skylea": {
+            name: "sky.lea.moe's Theme",
```
Comment (metalcupcake5) [inline src/constants/themes.js:33]:
```suggestion
            name: "sky.lea.moe",
```
imo "sky.lea.moe" sounds better than "sky.lea.moe's theme"

**Review by metalcupcake5 — CHANGES_REQUESTED** [review]
Looks good

**Review by metalcupcake5 — APPROVED** [review]
Looks good

**Review by boblovespi — CHANGES_REQUESTED** [review]
image
you can't select certain packs on mobile

**Review by boblovespi — APPROVED** [review]

**Pull request merged**